modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -519,11 +519,9 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
             $fieldlist[$fieldname]['value'] = $timedate->to_display_date_time($value, true, true);
         }
         $fieldlist[$fieldname]['name'] = $aow_field;
-    } else if(isset( $fieldlist[$fieldname]['type'] ) && ($fieldlist[$fieldname]['type'] == 'datetimecombo' || $fieldlist[$fieldname]['type'] == 'datetime' || $fieldlist[$fieldname]['type'] == 'date')){
+    } else if (isset($fieldlist[$fieldname]['type']) && ($fieldlist[$fieldname]['type'] == 'datetimecombo' || $fieldlist[$fieldname]['type'] == 'datetime' || $fieldlist[$fieldname]['type'] == 'date')) {
         $value = $focus->convertField($value, $fieldlist[$fieldname]);
-        $fieldlist[$fieldname]['value'] = $timedate->to_display_date($value);
-        //$fieldlist[$fieldname]['value'] = $timedate->to_display_date_time($value, true, true);
-        //$fieldlist[$fieldname]['value'] = $value;
+        $fieldlist[$fieldname]['value'] = substr($value, 0, 10);
```

Comment:
If you have to replace to_display_date() function to fixing this issue, it means something wrong with in this function? If it is the case may better practice to fixing it centralized.

Reply:
@gymad The to_display_date function converts DB date string to local date string but in this case we don't want to do any conversions.